File: resume-matching/src/main/java/com/ostj/dataentity/Job.java
package com.ostj.dataentity;

import java.util.Date;

public class Job {
    public int id;
    public String external_id;
    public String title;
    public String location_country;
    public String location_city;
    public String location_state;
    public Date published;
    public String description;
    public String apply_url;
    public float salary_min;
    public float salary_max;
    public String location_is_remote;
    public String type;
        
    public String toString(){
        return String.format("Id=%d", id )+", ext_id="+external_id+", title="+title+", location="+location_country+", published="+published+", application_url="+apply_url;
    }
}

File: OstjApp/OstjApi/Models/Job.cs
using Microsoft.EntityFrameworkCore;

namespace OstjApi.Models
{
    [Index(nameof(ext_id), IsUnique = false)]
    [Index(nameof(title), IsUnique = false)]
    [Index(nameof(location), IsUnique = false)]
    [Index(nameof(published), IsUnique = false)]
    [Index(nameof(remote), IsUnique = false)]
    [Index(nameof(type), IsUnique = false)]
    public class Job
    {
        [Key]
        public int Id;
        public String ext_id;
        public String title;
        public String location;
        public String published;
        public String description;
        public String application_url;
        public String salary;
        public String remote;
        public String type;
    }
}